docs/builtins.md

```diff
@@ -278,16 +321,18 @@ Writes `bytes`, a list of U24 with each element representing a byte, to the file
 
 Returns nothing (`*`).
 
-Writing discards any preexisting content that came after the current position. For example, if your file contains the text `Hello, world!` and the current position is at the `,`, writing `!` will 
+Writing discards any preexisting content that came after the current position. For example, if your file contains the text `Hello, world!` and the current position is at the `,`, writing `!` will
```

Comment:
```suggestion
```
I started to write this, tested and saw that it was not true, but forgot to erase this line.